data_steward/tools/run_curation_pipeline.sh

```diff
@@ -52,68 +52,82 @@ while true; do
-echo "unioned_ehr_dataset --> $unioned_ehr_dataset"
-"${TOOLS_DIR}/generate_unioned_ehr_dataset.sh" --key_file ${key_file} --ehr_snap_dataset ${ehr_snap_dataset} --vocab_dataset ${vocab_dataset}
+"${TOOLS_DIR}/generate_unioned_ehr_dataset.sh" --key_file ${key_file} --ehr_snap_dataset ${ehr_snapshot} --vocab_dataset ${vocab_dataset} --dataset_release_tag ${dataset_release_tag}
 
 #---------------------------------------------------------
 # Step 3 Generate combined dataset
 echo "-------------------------->Generate combined ehr rdr dataset (step 3)"
-cdr="combined${today}"
-tag=$(git describe --abbrev=0 --tags)
-version=${tag}
-echo "cdr --> $cdr"
-"${TOOLS_DIR}/generate_combined_dataset.sh" --key_file ${key_file} --vocab_dataset ${vocab_dataset} --unioned_ehr_dataset ${unioned_ehr_dataset} --rdr_dataset ${rdr_dataset}
+"${TOOLS_DIR}/generate_combined_dataset.sh" --key_file ${key_file} --vocab_dataset ${vocab_dataset} --unioned_ehr_dataset ${unioned_ehr_dataset} --rdr_dataset ${rdr_dataset} --dataset_release_tag ${dataset_release_tag}
 
 #-------------------------------------------------------
 # Step 4 Run achilles on combined dataset
 echo "-------------------------->Run achilles on identified CDR"
-export BIGQUERY_DATASET_ID="${cdr}"
+combined_dataset="${dataset_release_tag}_combined"
+combined_backup="${combined_dataset}_backup"
 export BUCKET_NAME_NYC="test-bucket"
-"${TOOLS_DIR}/run_achilles_report.sh" --dataset ${cdr} --key_file ${key_file}  --result_bucket ${result_bucket}
+
+# Running Achilles analysis on Combined dataset
+export BIGQUERY_DATASET_ID="${combined_backup}"
+"${TOOLS_DIR}/run_achilles_report.sh" --dataset ${combined_backup} --key_file ${key_file} --result_bucket ${result_bucket}
+
+# Running Achilles analysis on combined clean datased
+export BIGQUERY_DATASET_ID="${combined_dataset}"
+"${TOOLS_DIR}/run_achilles_report.sh" --dataset ${combined_dataset} --key_file ${key_file} --result_bucket ${result_bucket}
 
 #--------------------------------------------------------
 # Step 5 Run deid on cdr
 echo "-------------------------->Run de identification on the identified CDR"
-cdr_deid="${cdr}_deid"
+cdr_deid="${combined_dataset}_deid"
 echo "cdr_deid --> ${cdr_deid}"
-"${TOOLS_DIR}/deid_runner.sh" --key_file ${key_file} --cdr_id ${cdr} --vocab_dataset ${vocab_dataset}
+"${TOOLS_DIR}/deid_runner.sh" --key_file ${key_file} --cdr_id ${combined_dataset} --vocab_dataset ${vocab_dataset} --dataset_release_tag ${dataset_release_tag}
 
 #-------------------------------------------------------
 # Step 6 Run achilles on de-identified dataset
-echo "-------------------------->Run achilles on identified CDR"
-echo "cdr_deid --> ${cdr_deid}"
-"${TOOLS_DIR}/run_achilles_report.sh" --dataset ${cdr_deid} --key_file ${key_file} --result_bucket ${result_bucket}
+echo "-------------------------->Run achilles on de-identified CDR"
+cdr_deid_base="${cdr_deid}_base"
+cdr_deid_clean="${cdr_deid}_clean"
+export BUCKET_NAME_NYC="test-bucket"
```

Comment:
Is this line necessary?

Reply:
yes, it is a required environment variable. I never got to look deep into why it is required but will definitely look into it.

Reply:
Ok.  We should look into this, but that investigation shouldn't be part of this PR.  Leave it as is to make sure this executes correctly.

Reply:
I created a Jira issue in the backlog for this.  DC-651.